### DISCUSSION TITLE
"toptitle is not defined" bug

### DISCUSSION THREAD

**Discussion by sumingwang233** [windingwind/zotero-better-notes#583] [Q&A] [answered]
I can't ust most templates 'cause of the reason above

using newest edition1.0.4

(weird that the introdution of this edition clearly said this bug is dead, but anyway, I'm still dying

**Comment by sumingwang233** [comment]
"ust"---"use"*

apology for bad english

**Reply by QinCity** [reply, marked as answer]
You can try to change the template name like: `SCI template` to `[item]SCI template`

**Reply by sumingwang233** [reply]
ty
